ProcessLib/BoundaryCondition/BHEBottomDirichletBoundaryCondition.h

```diff
@@ -0,0 +1,115 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include "BoundaryCondition.h"
+#include "NumLib/DOF/LocalToGlobalIndexMap.h"
+#include "NumLib/IndexValueVector.h"
+#include "ProcessLib/HeatTransportBHE/BHE/BHEAbstract.h"
+#include "ProcessLib/Parameter/Parameter.h"
+
+namespace ProcessLib
+{
+class BHEBottomDirichletBoundaryCondition final : public BoundaryCondition
+{
+public:
+    BHEBottomDirichletBoundaryCondition(
+        GlobalIndexType global_idx_T_in_bottom,
+        GlobalIndexType global_idx_T_out_bottom,
+        MeshLib::Mesh const& bulk_mesh,
+        std::vector<MeshLib::Node*> const& vec_outflow_bc_nodes,
+        int const variable_id,
+        unsigned const integration_order,
+        std::size_t const bulk_mesh_id,
+        int const component_id,
+        unsigned const bhe_idx)
+        : _bulk_mesh_id(bulk_mesh_id), _bulk_mesh(bulk_mesh)
+    {
```

Comment:
It would be better to move the implementation in the cpp file.

Reply:
moved both constructor functions to cpp.